### DISCUSSION TITLE
posts/uwubernetes-kubernetes-v1-30-for-illumos-openbsd-freebsd/

### DISCUSSION THREAD

**Discussion by giscus** [tnorlin/kubernaut.eu#41] [Announcements]
# posts/uwubernetes-kubernetes-v1-30-for-illumos-openbsd-freebsd/

Kubernetes v1.30 was released yesterday and although I have explored some interesting features in the upcoming Cilium v1.16 release...

https://kubernaut.eu/posts/uwubernetes-kubernetes-v1-30-for-illumos-openbsd-freebsd/

**Comment by Martinfx** [comment]
Hello,
how to build from source code. I would like make port from your source code but i have problem with compilation.

**Reply by Martinfx** [reply]
but source code has date Aug 28, 2023

**Reply by tnorlin** [reply]
Hi, this is the first comment/reaction to any post since I began to publish them, and I've apparently missed this one, sorry.
I guess you've figured it out by now, but a git diff between the upstream tag and my branch give output to patch the source tree to be able to compile.
As for compilation I just do a `m̀ake` (on illumos, `gmake`) on each binary, i.e. 
```
gmake cmd/kubectl
```

I believe that the reason of source code being "old" is due to me not regularly synking the main branch of my fork.
